extensions-core/avro-extensions/pom.xml

```diff
@@ -57,30 +57,13 @@
           <groupId>org.slf4j</groupId>
           <artifactId>slf4j-api</artifactId>
         </exclusion>
-        <exclusion>
-          <groupId>org.codehaus.jackson</groupId>
-          <artifactId>jackson-core-asl</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>org.codehaus.jackson</groupId>
-          <artifactId>jackson-mapper-asl</artifactId>
-        </exclusion>
```

Comment:
I noticed that instead of these, `avro-mapred` version `1.9.0` includes `com.fasterxml.jackson.core` version `2.9.8`, will that cause any problems?

Reply:
Yes, it moved from Jackson 1.x `org.codehaus.jackson` to 2.x `com.fasterxml.jackson`. If this is a problem, it depends. I think for Druid itself it will pan out, but in combination with different Hadoop versions, as always, there might be issues.

I also have a PR to bump Druid to the same version: https://github.com/apache/incubator-druid/pull/7701.  As you can see there are no breaking API changes, so I don't expect a lot of issues.

Reply:
I'm fine with excluding the `com.fasterxml.jackson` from Apache Avro if you prefer that.

Reply:
Excluding here I think would be best if everything still works.

Reply:
Excluding is the safest path, I've added the exclusions 👍